src/styles/blocks/place.scss

```diff
@@ -0,0 +1,46 @@
+.place {
```

Comment:
```suggestion
.place {
  &__photos {
    margin-bottom: somepixels;
  }

```
you have no margin after `place__photos` block

![image](https://user-images.githubusercontent.com/55802630/157396576-97d7f473-2906-4388-b440-8bde7018ae93.png)

Reply:
Thanks, fixed all.